README.md

```diff
@@ -3,38 +3,78 @@
 [![Discord](https://img.shields.io/discord/310910488152375297?label=Discord)](http://phalcon.link/discord)
 [![codecov](https://codecov.io/gh/phalcon/migrations/branch/master/graph/badge.svg)](https://codecov.io/gh/phalcon/migrations)
 
-Generate or migrate database changes via migrations.
+Generate or migrate database changes via migrations.  
+Main idea of Phalcon migrations is to automatically detect changes and morphing without writing manual migrations.
+
+## Full documentation
+
+https://docs.phalcon.io/4.0/en/db-migrations
```

Comment:
Can we link to https://docs.phalcon.io/latest/en/db-migrations this will be the default after Seo changes.